stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/selectedmessage/SelectedReactionsMenu.kt

```diff
@@ -0,0 +1,151 @@
+package io.getstream.chat.android.compose.ui.components.selectedmessage
+
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.heightIn
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import io.getstream.chat.android.client.models.Message
+import io.getstream.chat.android.client.models.Reaction
+import io.getstream.chat.android.client.models.User
+import io.getstream.chat.android.common.state.MessageAction
+import io.getstream.chat.android.common.state.React
+import io.getstream.chat.android.compose.previewdata.PreviewReactionData
+import io.getstream.chat.android.compose.previewdata.PreviewUserData
+import io.getstream.chat.android.compose.state.userreactions.UserReactionItemState
+import io.getstream.chat.android.compose.ui.components.reactionoptions.ReactionOptions
+import io.getstream.chat.android.compose.ui.components.userreactions.UserReactions
+import io.getstream.chat.android.compose.ui.theme.ChatTheme
+
+/**
+ * Represents the list of user reactions.
+ *
+ * @param message The selected message.
+ * @param currentUser The currently logged in user.
+ * @param onMessageAction Handler that propagates click events on each item.
+ * @param modifier Modifier for styling.
+ * @param shape Changes the shape of [SelectedReactionsMenu].
+ * @param overlayColor The color applied to the overlay.
+ * @param reactionTypes The available reactions within the menu.
+ * @param onDismiss Handler called when the menu is dismissed.
+ * @param headerContent The content shown at the top of the [SelectedReactionsMenu] dialog. By default [ReactionOptions].
+ * @param bodyContent The content shown in the [SelectedReactionsMenu] dialog. By Default [UserReactions].
+ */
+@Composable
+public fun SelectedReactionsMenu(
```

Comment:
This component might warrant a small refactor to comply with our new API improvements [PR](https://github.com/GetStream/stream-chat-android/pull/2795).

Sister component `SelectedMessageOverlay` refactor seen [here](https://github.com/GetStream/stream-chat-android/blob/b015668bc25ee92c82a66a1c92e6cca39de81909/stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/selectedmessage/SelectedMessageMenu.kt#L48)

Reply:
Agreed - we could add the DefaultX components for the slots, make them internal and all.

Reply:
Done